package.json

```diff
@@ -32,8 +32,7 @@
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
     "express": "^4.12.0",
-    "loopback": "2.x.x",
-    "loopback-datasource-juggler": "2.x.x",
```

Comment:
My comment applies to a typical LoopBack project that calls `require('loopback')` and does not touch loopback's internals like juggler. Sorry for not making it more clear!

Most of the connectors are expected to work with vanilla juggler, therefore their tests should use juggler directly too. In which case `loopback` should not be listed in their (dev) dependencies at all.

AFAICT, `loopback` is not used in this connector at all (I run `grep "require(.loopback"` to verify).

That means we can go ahead, remove `loopback` and keep `loopback-datasource-juggler` in the dev dependencies only.